services/api/src/typeDefs.js

```diff
@@ -579,6 +579,7 @@ const typeDefs = gql`
     contentType: String
     notificationSeverityThreshold: ProblemSeverityRating
     organization: Int
+    pid: Int
```

Comment:
Are these `pid` used? It looks like they aren't, so can this be removed?

Reply:
Added the `pid` initially to be able to show assigned vs unassigned notifications, though this setup doesn't account for a notification assigned to multiple projects. Do you think it's worth adding that functionality? Being able to retreive a notification and all of its assigned projects? The response looking something like this:

![image](https://github.com/uselagoon/lagoon/assets/40746380/2f10bba1-c1e0-4c82-bd95-9af1759a4f35)

Reply:
Yes, but not like that.

It'd be better to use the `project: [Project]` instread of `pid: Int` and return the projects objects, but this is a more involved change.

Reply:
I think for now, remove `pid` and leave it the way it is.

Notifications are kinda painful association wise as they are

Reply:
I'd see something like `getProjectsByNotification` and the input would be the notification, and the response would be just the projects.

This way it wouldn't require any schema changes beyond a new query